frontend/src/components/home-page/components/filter-dialog.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from 'hooks/hooks';
+import Button from '../../common/button/button';
+import Checkbox from '../../common/checkbox/checkbox';
```

Comment:
```suggestion
import { Button, Checkbox } from 'components/common/button/button';
```

Reply:
I do not understand how this can be done, can you explain in more detail?

Reply:
```suggestion
import { Button, Checkbox } from 'components/common/common';
```
you need to use proxy-import